.github/workflows/ci-test.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         python -m pip install --upgrade pip wheel
         # TODO(b/232490018): Cython need to be installed separately to build pycocotools.
         python -m pip install Cython -c ./test_constraints.txt
-        TFX_DEPENDENCY_SELECTOR=NIGHTLY pip install -c ./test_constraints.txt --extra-index-url https://pypi-nightly.tensorflow.org/simple --pre --editable .[all]
+        TFX_DEPENDENCY_SELECTOR=NIGHTLY pip install -c ./test_constraints.txt --extra-index-url https://pypi-nightly.tensorflow.org/simple --pre .[all]
```

Comment:
I'm curious - why is editable no longer necessary? Anyway, this is a major improvement - thank you!

Reply:
It has to do with the building of wheels. When using `--editable`, no wheel is built. Instead, a symbolic link is created, pointing to your source code directory, allowing you to edit the code and see the changes instantly. The building of wheels is not recommended for `tfx-dev` package: https://github.com/tensorflow/tfx/blob/4012ecbe64d737ada5ae68ac8f8ce874bc23aa7a/setup.py#L76-L83

See https://github.com/tensorflow/tfx/blob/4012ecbe64d737ada5ae68ac8f8ce874bc23aa7a/package_build/README.md?plain=1#L49

Now we just skip building wheels (even without `--editable` flag), instead of raising error and stopping the installation altogether.